src/applications/model/gaming-streaming-server.cc

```diff
@@ -49,24 +48,22 @@ GamingStreamingServer::GetTypeId (void)
                    UintegerValue (100),
                    MakeUintegerAccessor (&GamingStreamingServer::m_peerPort),
                    MakeUintegerChecker<uint16_t> ())
-    // TODO: remove these
-    .AddAttribute ("InterArrivalTime", "A RandomVariableStream used to pick the inter-arrival time between packets",
-                   PointerValue (),
-                   MakePointerAccessor (&GamingStreamingServer::m_interArrivalTime),
-                   MakePointerChecker <RandomVariableStream>())
-    .AddAttribute ("PacketSize", "A RandomVariableStream used to pick the size of packets",
-                   PointerValue (),
-                     MakePointerAccessor (&GamingStreamingServer::m_packetSize),
-                     MakePointerChecker <RandomVariableStream>())
+    .AddTraceSource ("Tx", "A new packet is created and is sent",
+                     MakeTraceSourceAccessor (&GamingStreamingServer::m_txTrace),
+                     "ns3::Packet::TracedCallback")
   ;
   return tid;
 }
 
 GamingStreamingServer::GamingStreamingServer ()
-  : m_sentSeq (0),
+  : m_seq (0),
     m_totSentPackets (0),
+    m_totFailedPackets (0),
+    m_totSentBytes (0),
     m_socket (0),
-    m_sendEvent (EventId ())
+    m_peerAddress (Address ()),
+    m_peerPort (0),
+    m_trafficStreams (0, TrafficStream ())
```

Comment:
you don't need to initialize vectors: they are already empty-initialized by default